packages/dotcom-ui-header/src/__stories__/story.tsx

```diff
@@ -15,15 +14,6 @@ import profileStoryData from './story-data/profile'
 import '../../styles.scss'
 import './demos.scss'
 
-const toggleUserStateOptions = () => boolean('Enable user nav actions', true)
```

Comment:
The knobs version seemed to have way less duplication, I wonder if there's a better way to use Controls to avoid so much duplication?

Reply:
Perhaps we could have all the controls as part of the default export at the top.

And disabled them like so:

https://storybook.js.org/docs/react/essentials/controls#disable-controls-for-specific-properties

Reply:
One other argument would be that these components are different enough they should have their own stories entirely maybe?

Reply:
Simplified by disabling and using inferred booleans as per Kara's suggestion...